### PR TITLE
Removing The Term CloudWatch

### PR DESCRIPTION
**Pull request by SohailHaqyar**

### CODE REVIEW

File: src/models/deployment/remote-connect/vpn-users.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+interface IVpnUsers {
```
Comment (TonyH-Salient) [inline src/models/deployment/remote-connect/vpn-users.ts:1]: This should be in single form `IVpnUser`, shouldn't it?

File: src/models/deployment/remote-connect/proxy-logs.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+interface IFormattedProxyLogs {
```
Comment (TonyH-Salient) [inline src/models/deployment/remote-connect/proxy-logs.ts:1]: Here as well. Single form `IFormattedProxyLog`

File: src/models/deployment/remote-connect/proxy-logs.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  timestampRelative: string | null;
+  message: string;
+  json: string | null;
+}
+
+interface IProxyLogs {
+  Results: Array<Array<IProxyLogsResult>>;
+}
+
+interface IProxyLogsResult {
```
Comment (TonyH-Salient) [inline src/models/deployment/remote-connect/proxy-logs.ts:12]: This one too, single form `IProxyLogResult `

**Review by TonyH-Salient — CHANGES_REQUESTED** [review]

File: src/models/deployment/remote-connect/proxy-logs.ts (hunk 3 above)
Comment (SohailHaqyar) [reply src/models/deployment/remote-connect/proxy-logs.ts]: [26af6ef](https://github.com/SalientSys/salient-sys-cloud-dtos/pull/28/commits/26af6ef24cd52455019ded81fcd7a535b45de8f7)

File: src/models/deployment/remote-connect/proxy-logs.ts (hunk 2 above)
Comment (SohailHaqyar) [reply src/models/deployment/remote-connect/proxy-logs.ts]: [26af6ef](https://github.com/SalientSys/salient-sys-cloud-dtos/pull/28/commits/26af6ef24cd52455019ded81fcd7a535b45de8f7)

File: src/models/deployment/remote-connect/vpn-users.ts (hunk 1 above)
Comment (SohailHaqyar) [reply src/models/deployment/remote-connect/vpn-users.ts:1]: [26af6ef](https://github.com/SalientSys/salient-sys-cloud-dtos/pull/28/commits/26af6ef24cd52455019ded81fcd7a535b45de8f7)

**Pull request merged**